### ISSUE TITLE
Big exploit

### ISSUE THREAD

**Issue by kuncoft** [Zrips/Residence#274]
I revealed exploit. When set MoneyBack on true in config to give money back after residence removal, it give much more money back than residence's original cost. Im not ignoring Y coordinates, so I think it gives back money from complete Y coord.

**Comment by Zrips** [comment]
This can be due to some one selling residence to another player which has lower per block sell price then original player who created residence. As sell price is been determined by creators rank.
In general some examples would be nice to have as it should be working as intended. Residence worth is set by its creators group. Should we have option to update that worth value by current owner? Tho this could only going one way, decreasing worth of residence, otherwise we are risking to introduce exploits. 
In short, its not a exploit as residence creator paid more to create it then player who sold it.
